src/blameandshame/localization.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, List, Type, TypeVar
+from blameandshame.components import Component
+import yaml
+
+VERSION = "1.0"
+G = TypeVar('G', bound=Component)
+S = TypeVar('S', bound=Component)
+
+
+class Localization (object):
+
+    @staticmethod
+    def from_file(filename: str) -> 'Localization':
+        """
+        Builds a Localization object from a file
+        """
+        with open(filename) as f:
+            try:
+                loc = yaml.load(f)
+            except KeyError as err:
+                print('Error when importing', filename, '.', err)
+                raise IOError
+        return loc
```

Comment:
What is the type of `loc`? Unless I've misunderstood, `yaml.load(f)` should return a dictionary? Does `loc` not need to be unpacked into a `Localization` object?